### DISCUSSION TITLE
Cookbook launch.json then pick config

### DISCUSSION THREAD

**Discussion by duarm** [mfussenegger/nvim-dap#1275] [Ideas]
I've noticed that 99% of the time I run the debugger, I want to run the launch.json config. Is there way to "Run this config in this condition automatically", in my case, "run the launch.json if present, otherwise show the picker.", in order to save some clicks.

Thanks

**Comment by MartinDelille** [comment]
I had the same question. I tried to rename my configuration to `default` and it was executed without prompting. Can you try on your side ?

**Reply by MartinDelille** [reply]
Sorry for the bad advice: renaming the configuration to default has no effect...
